services/repository-managers/pom.xml

```diff
@@ -12,6 +12,20 @@
 
 	<dependencies>
 
+		<!-- For parsing Markdown -->
+		<dependency>
+			<!-- jsoup HTML parser library @ http://jsoup.org/ -->
+			<groupId>org.jsoup</groupId>
+			<artifactId>jsoup</artifactId>
+			<version>1.7.3</version>
+		</dependency>
+
+		<dependency>
+			<groupId>org.sagebionetworks</groupId>
+			<artifactId>markdown</artifactId>
+			<version>2015-06-22-6486f39-16</version>

```

Comment:
Not sure how to check this unless my build fails when I try a final build before again asking for a code review.